modules/otel-collector-components.adoc

```diff
@@ -869,7 +902,47 @@ This extension supports traces, metrics, and logs.
 <10> The path to the client's private key file that is used with the client certificate if needed for authentication.
 <11> Sets a timeout for the token client's request.
 <12> You can assign the authenticator configuration to an OTLP exporter.
+[id="filestorage-extension_{context}"]
+=== Filestorage extension
+
+The File Storage extension can persist state to the local file system.
+The extension requires read and write access to a directory. A default directory can be used, but it must already exist in order for the extension to operate.
+
+The OAuth2Client extension persists the sending queue for the OTLP exporters that are based on the HTTP and the gRPC protocol.
```

Comment:
@frzifus, is  `OAuth2Client` mentioned here intentionally or is this just a leftover from the copy-and-paste steps when preparing this draft?

Reply:
Ups, thats a leftover from copy pasta.

```suggestion
```